app/models/report.rb

```diff
@@ -51,6 +51,14 @@ def destroy_reported_item
     mark_as_reviewed
   end
 
+  def mark_as_nsfw
```

Comment:
> How about either ignoring whiny requests

I do that, but I decided for myself that NSFW is something which bugs me too.
And this is something you can decided for yourself I just implemented a option

> talking to the user or setting the profile NSFW

if you have 10 reports from the same user that is probably the better solution,
but if not you can spend either contacting ten different people or mark it quickly as nsfw

>   I'm really not a fan of your "well, I'll just change it locally and don't care about the rest of the world" solution here.

Never said that. Give me a federated-solution and I change it right away